testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1120,6 +1120,75 @@ QUnit.test("searchEditorOptions", function(assert) {
     assert.strictEqual(searchEditorInstance.option("placeholder"), "Test", "placeholder of the search box");
 });
 
+QUnit.test("apply list search options if dataSource set as dataSource instance", function(assert) {
+    var data = [{ expr: "test1" }, { expr: "2test" }],
```

Comment:
`data` is not required. You can use dataSource without options because the goal is to check its options only

`store: []`